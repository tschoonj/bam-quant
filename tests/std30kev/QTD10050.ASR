File: C:\xrfdaten\2014KW30\chris\std30kev\std10050.spe
Start: 22.07.2014 10:20:00
Stop: 22.07.2014 10:21:40
BESSY: 298.5215676 298.5609973
IONI: 1.954881E-11 0.000000000019525
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 68960
Run: 100000
Dead: 31
ICR: 103020
OCR: 0
Accumulated BESSY: 59703.8387441
Accumulated Ioni: 3.90845006E-09
N Accumulation: 200
Normfactor:  7.4204147e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Sn
exp_x 152.720397949219 152.7204
exp_y 92.4657974243164 92.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      2
  50   1     25.191   5418283.      2328.     102.01
  50   3     28.472    816332.       832.      34.29
