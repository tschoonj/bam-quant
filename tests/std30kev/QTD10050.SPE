$DATA:
0 2000
15619
10584
5337
1935
584
158
70
31
17
8
12
3
6
2
4
29
293
3408
4973
4899
4966
4828
4681
4593
4488
4370
4293
4263
4198
4135
4045
3978
3834
3928
3762
3746
3562
3558
3690
3605
3473
3433
3267
3318
3228
3182
3217
3083
3169
3181
3250
3085
3069
3021
3068
2961
3116
2967
2934
2849
2915
2913
2893
2892
2919
3023
2950
2959
3039
2949
2999
3027
3108
3145
3152
3060
3006
3027
3027
2987
3089
3144
3149
3210
3066
3096
3143
3173
3300
3290
3290
3356
3437
3431
3605
3483
3596
3651
3464
3543
3650
3608
3574
3553
3521
3360
3313
3224
3169
2977
2942
2792
2811
2759
2555
2527
2374
2345
2319
2181
2118
2090
1987
2015
1760
1868
1779
1763
1738
1725
1608
1673
1611
1612
1572
1435
1427
1414
1424
1405
1343
1282
1207
1231
1208
1259
1161
1075
1082
1008
1021
1040
931
914
904
841
867
819
838
836
773
794
695
719
702
710
694
694
735
652
645
625
639
625
604
605
594
596
608
571
585
567
556
526
529
524
509
516
515
490
479
467
496
494
453
466
494
465
466
505
436
453
456
438
409
421
393
421
409
413
435
404
416
394
393
378
379
378
425
399
389
385
373
350
382
361
389
382
365
359
375
358
366
328
338
343
343
366
345
374
323
343
396
304
332
360
322
293
310
344
343
304
371
336
318
369
324
343
339
338
326
338
293
345
363
378
401
442
459
417
427
362
367
324
314
342
300
283
287
292
298
313
283
310
273
272
291
290
285
285
337
280
295
338
309
293
313
309
289
309
281
237
312
268
263
265
277
303
269
275
285
358
421
554
596
687
754
784
790
659
605
498
402
322
313
271
247
253
269
270
260
272
256
257
247
238
235
255
249
270
266
257
257
259
289
275
321
339
320
306
295
315
309
289
267
263
264
269
242
249
240
259
236
239
257
291
275
282
280
258
288
288
280
266
291
237
243
242
244
256
241
241
245
240
271
233
250
212
231
242
216
241
233
250
232
262
258
259
271
248
245
222
246
249
216
251
235
253
242
238
238
240
246
233
242
200
223
260
246
248
225
216
227
236
216
249
222
219
249
200
231
212
245
217
234
214
248
233
221
240
212
246
212
247
232
245
221
226
203
224
234
190
242
248
252
235
242
250
281
277
262
261
259
239
229
217
233
253
244
215
200
209
227
223
223
206
204
237
232
232
219
199
209
204
212
208
194
215
216
238
202
199
195
199
191
208
217
186
218
222
221
233
215
208
195
216
230
230
231
218
226
210
210
212
219
201
226
201
252
215
243
218
239
192
252
237
211
207
212
190
220
204
187
207
214
253
234
242
229
222
222
224
200
201
213
197
200
235
205
225
200
215
201
214
177
209
180
202
204
206
200
209
209
212
215
212
189
210
189
203
201
202
207
195
201
196
202
185
201
199
209
191
209
183
175
184
219
209
197
204
197
219
204
187
210
209
205
180
178
192
207
204
197
193
174
179
187
206
212
201
180
212
200
196
201
194
193
182
205
208
243
231
211
234
252
240
256
250
213
219
224
197
212
214
211
181
188
166
198
191
226
247
238
241
246
246
256
244
218
232
181
214
205
192
204
213
203
194
203
200
218
182
204
167
190
194
173
172
197
196
224
197
184
184
223
191
161
187
172
202
217
172
225
176
184
183
218
197
198
205
211
206
217
229
233
196
209
210
214
202
228
211
206
215
225
219
222
217
191
189
192
189
216
208
207
194
208
194
199
236
218
214
195
209
219
218
217
229
249
233
237
252
247
269
248
269
259
279
236
281
254
297
259
275
277
263
276
294
305
271
312
288
284
317
332
298
291
316
299
314
325
348
322
385
419
525
681
1021
1455
1923
2985
4028
5753
7664
9652
12101
14172
16307
17915
18965
19323
18982
17511
15733
13559
11146
8914
6703
4917
3357
2298
1478
1075
731
499
395
301
308
261
229
247
248
224
202
199
227
194
198
224
223
214
231
192
192
196
227
207
198
210
198
188
218
210
210
219
217
220
207
224
203
222
227
218
182
202
221
197
211
207
172
228
205
211
210
221
243
241
229
239
216
178
208
217
216
237
267
255
244
279
277
300
333
375
442
525
660
818
1078
1376
1649
2002
2462
2836
3070
3112
3225
3076
2804
2575
2150
1722
1483
1251
981
880
743
656
670
654
646
606
550
541
495
469
445
348
316
255
236
225
222
195
198
179
204
181
169
190
170
217
196
182
169
203
212
191
209
194
180
200
184
178
201
196
203
201
196
206
218
208
179
206
204
208
189
185
174
226
201
192
207
221
222
192
188
208
217
214
210
195
212
223
192
206
177
190
208
211
218
200
202
223
230
234
223
232
232
199
187
248
221
217
226
209
219
238
209
223
207
221
222
200
220
202
242
186
230
232
237
221
212
242
230
242
240
251
213
253
258
246
249
257
256
232
251
237
231
235
267
274
252
249
270
242
267
241
281
255
274
242
284
274
280
289
282
290
283
280
296
295
296
301
304
305
289
252
283
297
294
339
339
280
296
309
337
303
282
327
297
306
298
341
352
315
320
357
311
306
318
291
336
335
362
333
350
336
360
388
329
342
366
348
371
362
408
411
402
386
418
433
455
417
416
420
444
457
476
476
484
449
498
497
488
504
512
520
489
542
578
511
560
567
579
571
584
621
630
626
634
700
680
705
696
740
768
778
770
796
816
814
909
934
1069
973
999
990
1080
1137
1160
1177
1205
1267
1204
1288
1288
1318
1408
1430
1430
1485
1491
1524
1606
1589
1591
1689
1699
1712
1775
1754
1788
1784
1812
1787
1812
1812
1896
1843
1820
1816
1883
1819
1848
1895
1790
1806
1726
1741
1705
1765
1662
1629
1600
1554
1514
1514
1495
1438
1472
1460
1419
1508
1487
1515
1472
1487
1504
1456
1513
1418
1563
1501
1550
1505
1574
1583
1636
1654
1690
1705
1634
1716
1709
1859
1863
1827
1873
2015
2018
2151
2116
2216
2248
2260
2356
2482
2563
2746
2900
3191
3574
3985
4560
5522
7080
9173
12342
16988
22896
30536
40375
52310
66210
82037
98325
114646
129946
145010
158298
170144
181405
190302
200219
210677
222031
234549
248083
257987
266390
269873
267648
260198
244515
223918
199406
172204
144188
117253
92811
71140
53537
38528
27646
19385
13433
9370
6753
4972
3526
2941
2390
2009
1781
1635
1544
1455
1317
1255
1257
1177
1039
1079
998
907
960
886
854
912
827
787
813
793
787
754
713
752
761
708
674
663
649
627
608
627
645
647
619
614
585
609
569
537
567
554
569
560
573
526
523
521
548
510
524
524
519
547
515
565
516
510
544
508
491
469
503
507
505
472
482
496
517
517
519
482
506
506
502
506
532
520
510
525
558
490
507
548
531
570
584
579
531
554
542
604
571
558
574
563
612
626
611
602
568
623
616
609
578
666
690
645
726
740
724
721
798
788
915
893
1030
1136
1328
1595
2006
2628
3597
4933
6483
8738
11594
15254
19343
23882
28653
34467
39373
44397
48280
51195
52952
52840
51674
49188
45413
40849
35538
30091
25124
20434
15707
12222
9289
7140
5272
3892
3065
2616
2214
2211
2404
2686
3144
3836
4584
5440
6493
7338
8194
9008
9497
10058
10158
9857
9330
8763
7944
6806
5936
4910
4083
3177
2484
1928
1338
964
716
489
348
217
184
137
87
88
91
75
62
62
61
44
42
52
54
41
49
40
42
53
35
46
29
29
38
38
36
36
35
31
27
32
42
34
31
34
26
23
36
33
30
41
33
35
32
30
22
26
35
46
31
28
24
34
31
25
36
22
29
38
29
29
26
29
29
23
26
24
27
22
25
27
21
18
17
28
20
21
21
27
19
16
21
31
23
14
19
23
19
18
20
27
13
17
25
12
18
23
23
15
24
20
20
27
17
18
16
18
23
21
19
16
21
22
13
7
13
10
13
10
13
7
7
11
16
11
17
12
12
7
9
6
18
16
9
9
10
10
14
15
12
11
16
11
7
12
10
9
9
12
13
11
10
12
8
10
7
6
10
8
16
12
19
12
8
14
12
9
16
10
12
5
7
16
10
12
7
14
10
10
7
10
13
8
13
16
12
15
15
14
12
11
11
13
14
11
12
15
13
10
15
8
10
12
8
8
11
9
12
8
15
9
6
11
14
10
10
15
15
13
13
13
12
23
18
11
10
19
12
19
10
13
9
12
18
12
5
18
11
10
15
14
20
15
14
7
17
13
16
9
15
10
12
16
13
11
17
4
15
15
12
14
12
11
14
18
17
13
16
11
24
29
22
18
17
23
25
18
27
19
14
26
17
21
23
14
16
18
21
18
19
20
20
21
18
10
15
8
11
13
7
10
15
11
12
10
10
9
5
7
7
7
3
5
6
7
5
3
11
3
3
2
2
6
10
2
2
2
4
5
1
5
1
7
3
2
2
3
5
5
7
4
5
1
5
2
3
5
4
6
5
4
3
2
2
4
1
6
6
3
3
5
4
1
3
4
5
1
1
1
5
7
5
3
2
4
3
4
3
4
5
2
1
4
2
5
9
1
2
3
5
8
8
4
3
2
6
4
4
6
5
5
2
4
1
6
3
1
6
3
2
4
3
5
3
5
6
6
7
4
6
9
3
6
4
1
4
5
3
4
9
2
2
2
11
4
7
3
3
5
2
2
4
3
5
4
10
3
7
4
11
7
4
4
1
7
2
3
7
5
1
3
4
5
7
3
7
8
6
2
5
2
9
8
2
7
5
4
5
6
4
4
6
6
10
4
2
4
7
6
5
5
1
5
4
7
1
8
4
8
10
3
6
8
4
4
2
1
9
3
5
8
1
6
5
2
3
2
8
6
6
6
2
1
6
7
6
5
4
6
5
5
5
4
10
4
4
12
8
10
13
16
19
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10050.spe
Start: 22.07.2014 10:20:00
Stop: 22.07.2014 10:21:40
BESSY: 298.5215676 298.5609973
IONI: 1.954881E-11 0.000000000019525
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 68960
Run: 100000
Dead: 31
ICR: 103020
OCR: 0
Accumulated BESSY: 59703.8387441
Accumulated Ioni: 3.90845006E-09
N Accumulation: 200
Normfactor:  7.4204147e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Sn
 
exp_x 152.720397949219 152.7204
exp_y 92.4657974243164 92.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      2
  50   1     25.191   5418283.      2328.     102.01
  50   3     28.472    816332.       832.      34.29
