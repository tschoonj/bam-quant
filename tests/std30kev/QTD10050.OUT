     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:55  บ   
     บ  Spectrum: QTD10050.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        3 iterations done    บ   
     บ  ChiSquare =   347.2   last change =   .01%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -16.0 ๑   100.0   บ     -16.0 ๑     1.1   บ   
     บ  GAIN (eV/ch)  บ     20.197 ๑   2.000   บ    20.197 ๑    .001   บ   
     บ det NOISE (eV) บ      134.3 ๑    40.0   บ     134.3 ๑     6.0   บ   
     บ  FANO factor   บ       .112 ๑    .050   บ      .112 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Sn-Ka                           5418283. ๑   2328.           บ   
     บ    KA1        25.271    .64942     3518755. ๑   1512.    138.7  บ   
     บ             1252.025    279.69        1413.           7.95E-01  บ   
     บ    KA2        25.044    .35058     1899528. ๑    817.     66.1  บ   
     บ             1240.786    278.72        1462.           8.03E-01  บ   
     บ                                                                 บ   
     บ  2 Sn-Kb                            964899. ๑    983.           บ   
     บ    KB1        28.486    .55780      538224. ๑    549.     34.1  บ   
     บ             1411.209    293.06         901.           6.80E-01  บ   
     บ    KB3        28.444    .28822      278108. ๑    285.     33.8  บ   
     บ             1409.129    292.89         908.           6.81E-01  บ   
     บ    KB2        29.109    .15397      148568. ๑    154.     33.4  บ   
     บ             1442.055    295.58         796.           6.59E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
